optimum/onnxruntime/trainer.py

```diff
@@ -1194,7 +1229,7 @@ def compute_loss_ort(self, model, inputs, input_names, output_names, return_outp
 
         input_feed = dict(map(lambda input_name: (input_name, inputs[input_name].cpu().numpy()), input_names))
         outputs = self.infer_sess.run(output_names, input_feed)
-        outputs[0] = torch.Tensor(outputs[0])  # logits: `numpy.ndarray` -> tensor
+        outputs[0] = torch.Tensor(outputs[0])
```

Comment:
Shouldn't you do this on the loss as [here](https://github.com/huggingface/optimum/blob/ort-train-11-compatibility/optimum/onnxruntime/trainer.py#L1242) you leave the possibility of outputs being a dictionnary

Reply:
Actually, the function `compute_loss_ort` will only be used for the ORT circumstance, otherwise `compute_loss` will be used. Thus the case of `outputs` being a dictionary does not really exist in that case. 
Indeed it is confusing because I didn't remove that cases of  `outputs` being a dictionary just 10 lines below...

Reply:
I think it would make sense to remove this case then, what do you think ?

Reply:
Yep, done in the new commit!